scripts/generate_syncset.py

```diff
@@ -1,16 +1,10 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # logic for sorted output: https://gist.github.com/oglops/c70fb69eef42d40bed06
 
-try:
-        # for python newer than 2.7
-    from collections import OrderedDict
-except ImportError:
-        # use backport from pypi
-    from ordereddict import OrderedDict
+from collections import OrderedDict
 
 import yaml
```

Comment:
Maybe we should use oyaml for determinism?

```suggestion
import oyaml as yaml
```

Reply:
TIL + Reading about it - yes, and thanks for that suggestion!